[2023-03-29T07:21:58.297+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: docker_sample.print_date scheduled__2023-03-27T20:20:00+00:00 [queued]>
[2023-03-29T07:21:58.303+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: docker_sample.print_date scheduled__2023-03-27T20:20:00+00:00 [queued]>
[2023-03-29T07:21:58.303+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2023-03-29T07:21:58.303+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 2
[2023-03-29T07:21:58.303+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2023-03-29T07:21:58.326+0000] {taskinstance.py:1304} INFO - Executing <Task(BashOperator): print_date> on 2023-03-27 20:20:00+00:00
[2023-03-29T07:21:58.329+0000] {standard_task_runner.py:55} INFO - Started process 18354 to run task
[2023-03-29T07:21:58.334+0000] {standard_task_runner.py:82} INFO - Running: ['airflow', 'tasks', 'run', 'docker_sample', 'print_date', 'scheduled__2023-03-27T20:20:00+00:00', '--job-id', '599', '--raw', '--subdir', 'DAGS_FOLDER/test_docker_operator.py', '--cfg-path', '/tmp/tmpnn7rlkvn']
[2023-03-29T07:21:58.336+0000] {standard_task_runner.py:83} INFO - Job 599: Subtask print_date
[2023-03-29T07:21:58.398+0000] {task_command.py:389} INFO - Running <TaskInstance: docker_sample.print_date scheduled__2023-03-27T20:20:00+00:00 [running]> on host vibrdairflow.tciw5gv3lhjetcyg0kqzfvdylc.hx.internal.cloudapp.net
[2023-03-29T07:21:58.481+0000] {taskinstance.py:1511} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=airflow@example.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=docker_sample
AIRFLOW_CTX_TASK_ID=print_date
AIRFLOW_CTX_EXECUTION_DATE=2023-03-27T20:20:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-03-27T20:20:00+00:00
[2023-03-29T07:21:58.483+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-03-29T07:21:58.484+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'date']
[2023-03-29T07:21:58.493+0000] {subprocess.py:86} INFO - Output:
[2023-03-29T07:21:58.497+0000] {subprocess.py:93} INFO - Wed Mar 29 07:21:58 UTC 2023
[2023-03-29T07:21:58.497+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-03-29T07:21:58.528+0000] {taskinstance.py:1322} INFO - Marking task as SUCCESS. dag_id=docker_sample, task_id=print_date, execution_date=20230327T202000, start_date=20230329T072158, end_date=20230329T072158
[2023-03-29T07:21:58.586+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2023-03-29T07:21:58.615+0000] {taskinstance.py:2582} INFO - 2 downstream tasks scheduled from follow-on schedule check
